impli.cabal

```diff
@@ -1,4 +1,4 @@
-cabal-version: 3.12
+cabal-version: 3.8
```

Comment:
The cabal2nix from the pinned nixpkgs is aparently too old and was complaining about not recognizing 3.12
We can revert this in the near future after bumping the nixpkgs pin

Reply:
Not sure yet, will check

Reply:
> How to know when that is? 

Now